recipes/sas_kernel/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "SAS_kernel" %}
+{%set version = "2.1.7" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "730608f28beda4ed4d46f1ce9d3ef5269211e838d0cc5922a56427fdeaa935fc" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] | lower }}/{{ name | lower }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: NO_KERNEL_INSTALL=1 python setup.py install --single-version-externally-managed --record=record.txt
+  skip: true  # [not linux]
+  skip: true  # [py2k]
+
+requirements:
+  build:
+    - python 
+    - setuptools
+    - ipython >=4
+    - jupyter
+  run:
+    - python
+    - ipython >=4
+    - saspy >=1.2.2
+    - pygments
+    - metakernel >=0.18.0
+    - jupyter
```

Comment:
I don't think `jupyter` is required because `jupyter` brings the notebook, qtconsole and all their dependencies.

Reply:
Yes, I think you are correct it just needs `jupyter_client`.